static/templates/user_presence_row.handlebars

```diff
@@ -1,11 +1,10 @@
 <li data-user-id="{{user_id}}" class="user_sidebar_entry {{#if num_unread}} user-with-count {{/if}} narrow-filter {{#if faded}} user-fade {{/if}}">
     <div class="selectable_sidebar_block">
-        <span class="{{user_circle_class}} user_circle"></span>
+        <span class="{{user_circle_class}} user_circle user_circle_buddy" data-hover="{{user_circle_status}}"></span>
```

Comment:
Why do we need `data-hover` here?  If we're going to build the tooltip HTML when we hover this, then all we need is `data-user-id`, which is already available in the parent.

Reply:
This is for the user_circle which do not have complex data to display. The `data-hover` are supposed to work in conjunction with the more generic `register_tooltip` function where we can supply the pre-computed hover text directly. 
I left this as is because a lot of the other html title hovers we have right now can be changed this way.